### ISSUE TITLE
Sale post API

### ISSUE THREAD

**Issue by montanaday** [KyleAMathews/facebook-export#19]
Facebook groups have a specific feature - Sale Post that allows to post items for sale. What about filtering returned items so we only get for sale posts or even how to check if the post is a for sale post or not in a Group?  Thanks

**Comment by KyleAMathews** [comment]
Happy to take a PR that adds exporting sale posts (if they're not already exported) but not interested in adding support for filtering. You can get a JSON version of your posts which is easy enough to filter.

**Comment by montanaday** [comment]
Yes, the sale posts are exported but along with all other posts in a group. What I mean is to add an API call to check  if the posts are sale posts and then export them accordingly to JSON using your script.. .

**Comment by KyleAMathews** [comment]
If it's a non-disruptive change, happy to merge a PR!

**Comment by montanaday** [comment]
`
[

    {
        "from": {
            "name": "FB group user name",
            "id": "xxxxxxxxxxxxxxxxx"
        },
        "message": "Sale Item Header Text Here\n€50 - Berlin, Germany\n\nDetailed Item Description Text.",
        "picture": "https://scontent.xx.fbcdn.net/v/t1.0-0/......",
        "link": "https://www.facebook.com/photo.php.....",
        "created_time": "2016-09-19T15:38:34+0000",
        "updated_time": "2016-09-19T15:38:35+0000",
        "id": "xxxxxxxxxxxxxxxxx",
        "to": {
            "data": [
                {
                    "id": "xxxxxxxxxxx",
                    "name": "Item Name"
                }
            ]
        }
    }
]
`

**Comment by montanaday** [comment]
Here is the scope. As I  can see, there is not much for identifying a sale post on Facebook API. 

`
[

    {
        "from": {
            "name": "FB group user name",
            "id": "xxxxxxxxxxxxxxxxx"
        },
        "message": "Sale Item Header Text Here\n€50 - Berlin, Germany\n\nDetailed Item Description Text.",
        "picture": "https://scontent.xx.fbcdn.net/v/t1.0-0/......",
        "link": "https://www.facebook.com/photo.php.....",
        "created_time": "2016-09-19T15:38:34+0000",
        "updated_time": "2016-09-19T15:38:35+0000",
        "id": "xxxxxxxxxxxxxxxxx",
        "to": {
            "data": [
                {
                    "id": "xxxxxxxxxxx",
                    "name": "Item Name"
                }
            ]
        }
    }
]
`

**Comment by itpao25** [comment]
is there any news? I'm trying to get the information on sale posts

**Comment by andygomezb7** [comment]
You can only know why in the permalink url comes out sale_post_id but nothing else.

**Comment by ShaneRich5** [comment]
Is there a better way besides checking the permalink and parsing it yourself?
